p2p/test_util.go

```diff
@@ -8,7 +8,7 @@ import (
 	cmn "github.com/tendermint/tmlibs/common"
 	"github.com/tendermint/tmlibs/log"
 
-	cfg "github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/config"
```

Comment:
third change of this sort on my memory. we should make a standard and stick with it

Reply:
I made it consistent inside of `p2p` in the follow-up PR which comes right after this one is through.